test/integration/features/basic.feature

```diff
@@ -102,15 +102,6 @@ Feature: Basic
       hello
       """
 
```

Comment:
@praveenkumar If I really wanted to be picky, I would request even a different commit. so a revert would be much easier... ;-)